packages/manager/apps/pci-private-registry/src/constants.ts

```diff
@@ -1,3 +1,16 @@
+export const PRIVATE_REGISTRY_STATUS_MAP = {
+  ERROR: 'error',
+  READY: 'success',
+  DELETED: 'error',
+  SUSPENDED: 'error',
+  INSTALLING: 'info',
+  UPDATING: 'warning',
+  RESTORING: 'warning',
+  SUSPENDING: 'warning',
+  DELETING: 'warning',
+  SCALING_UP: 'info',
+};
+
```

Comment:
This variable is not used, and i think u dont need it, you can use the `RegistryStatus` component i have made before